clients/src/main/java/org/apache/kafka/clients/admin/MemberDescription.java

```diff
@@ -43,6 +43,12 @@
             new MemberAssignment(Collections.emptySet()) : assignment;
     }
 
+    public MemberDescription(String memberId,
```

Comment:
Hmm.. I noticed that `equals` and `hashCode` below do not account for `groupInstanceId`. Should they? It would be odd to include clientId, but not the group instance id.

Reply:
Agreed.